Logger/LogToSingleFileProcessor.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\GoogleAnalyticsImporter\Logger;
+
+
+use Monolog\Logger;
+use Piwik\Container\StaticContainer;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class LogToSingleFileProcessor
+{
+    /**
+     * @var string
+     */
+    public static $cliOutputPrefix = '';
+
+    private static $logToSingleFileHandled = false;
+
+    /** @var int $idSite */
+    private $idSite;
+
+    public function __construct($idSite)
+    {
+        $this->idSite = (int) $idSite;
+    }
+
+    public function __invoke(array $record)
+    {
+        $message = $record['message'];
+
+        if (is_string($message)) {
+            $record['message'] = '(idSite: ' . $this->idSite . ') ' . $record['message'];
```

Comment:
Not sure if it's done automatically by some other processor. If not be great to prefix `SettingsPiwik::getPiwikInstanceId`

Reply:
Looks like this is done by the cloud plugin. I'll merge w/o it unless you want it here as well.